invenio/modules/editor/bundles.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Editor bundles."""
+
+from invenio.ext.assets import Bundle
+
+js = Bundle(
+    "vendors/jquery/dist/jquery.js",
```

Comment:
I did it because being a legacy module I prefer to have full control over what libraries are in the page and avoid conflicts.

Currently the jquery bundle https://github.com/inveniosoftware/invenio/blob/pu/invenio/base/static/js/init.js includes too many libraries (like Bootstrap, typeahead) that are not needed in this page and could provoke some conflict.